crossdc-producer/src/main/java/org/apache/solr/update/processor/MirroringUpdateRequestProcessorFactory.java

```diff
@@ -152,6 +149,22 @@ public void inform(SolrCore core) {
 
                 KafkaCrossDcConf.readZkProps(properties, zkProps);
             }
+            CollectionProperties cp = new CollectionProperties(core.getCoreContainer().getZkController().getZkClient());
+             Map<String,String> collectionProperties = cp.getCollectionProperties(core.getCoreDescriptor().getCollectionName());
+            for (ConfigProperty configKey : KafkaCrossDcConf.CONFIG_PROPERTIES) {
+                String val = collectionProperties.get("crossdc." + configKey.getKey());
+                if (val != null && !val.isBlank()) {
+                    properties.put(configKey.getKey(), val);
+                }
+            }
+            String enabledVal = collectionProperties.get("crossdc.enabled");
```

Comment:
[0] We should probably document these special collection properties in Javadocs at the least, right?